rpc-client/src/subcommands/transactions_subcommands.rs

```diff
@@ -0,0 +1,751 @@
+use clap::{Args, Parser};
+use nimiq_keys::Address;
+use nimiq_primitives::coin::Coin;
+use nimiq_rpc_interface::{
+    consensus::ConsensusInterface,
+    types::{HashAlgorithm, ValidityStartHeight},
+};
+use nimiq_transaction::account::htlc_contract::AnyHash;
+
+use anyhow::Error;
+use async_trait::async_trait;
+
+use crate::Client;
+
+use super::accounts_subcommands::HandleSubcommand;
+
+#[derive(Debug, Args)]
+pub struct TxCommon {
+    /// The associated transaction fee to be payed. If absent it defaults to 0 NIM.
+    #[clap(short, long, default_value = "0")]
+    pub fee: Coin,
+
+    /// The block height from which on the transaction could be applied. The maximum amount of blocks the transaction is valid for
+    /// is specified in `TRANSACTION_VALIDITY_WINDOW`.
+    /// If absent it defaults to the current block height at time of processing.
+    #[clap(short, long, default_value_t)]
+    pub validity_start_height: ValidityStartHeight,
+
+    /// Don't actually send the transaction, but output the transaction as hex string.
+    #[clap(long)]
+    pub dry: bool,
+}
+
+#[derive(Debug, Args)]
+pub struct TxCommonWithValue {
+    /// The amount of NIM to be used by the transaction.
+    #[clap(long)]
+    pub value: Coin,
+
+    #[clap(flatten)]
+    pub common_tx_fields: TxCommon,
+}
+
+#[derive(Debug, Parser)]
+pub enum TransactionCommand {
+    /// Sends a simple transaction from the wallet `wallet` to a basic `recipient`.
+    Basic {
+        /// Transaction will be sent from this address. The sender wallet must be unlocked prior to this action.
+        #[clap(long)]
+        sender_wallet: Address,
+
+        /// Recipient for this transaction. This must be a basic account.
+        #[clap(short, long)]
+        recipient: Address,
+
+        #[clap(flatten)]
+        tx_commons: TxCommonWithValue,
+    },
+
+    /** Staker transactions **/
```

Comment:
```suggestion
    /* Staker transactions */
```
Otherwise it will appear in the clap help.